module #2. REST API Basics/rest-api-basic/core/src/main/java/com/epam/esm/configuration/AppConfig.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.esm.configuration;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan("com.epam.esm")
+public class AppConfig implements WebMvcConfigurer {
+}
```

Comment:
Please, leave a java doc comment for what configuration this class is used for. What is it purpose?